### PR TITLE
Resolve flake8 errors in alembic directory

### PR DESCRIPTION
**Pull request by yiboliu**
### Reasons for making this change
Resolve flake8 errors in alembic directory 

### Related issues
#2660 

### Checklist

* [ ] I've added a screenshot of the changes, if this is a frontend change
* [ ] I've added and/or updated tests, if this is a backend change
* [x] I've run the [pre-commit.sh](https://github.com/codalab/codalab-worksheets/blob/master/pre-commit.sh) script
* [ ] I've updated [docs](https://github.com/codalab/codalab-worksheets/tree/master/docs), if needed

### CODE REVIEW

File: alembic/versions/2014082017_add_worker_action_e_g__558ac277c2b9.py [hunk 1]
```diff
@@ -6,22 +6,22 @@
...
 revision = '558ac277c2b9'
 down_revision = '341ee10697f1'
 
-from alembic import op
-import sqlalchemy as sa
-
 
 def upgrade():
     pass
-    #op.create_table('bundle_action',
```
Comment (teetone) [inline alembic/versions/2014082017_add_worker_action_e_g__558ac277c2b9.py:19]: Can we just get rid of commented out code?

File: alembic/versions/2020081018_added_group_uuid_to_worker_9d3ab0b9d304.py [hunk 2]
```diff
@@ -6,13 +6,13 @@
```
Comment (teetone) [inline alembic/versions/2020081018_added_group_uuid_to_worker_9d3ab0b9d304.py:6]: Is there a way to hook up alembic with flake8 so we don't have to manually make these modifications?

File: docs/REST-API-Reference.md [hunk 3]
```diff
@@ -9,26 +9,14 @@ subject to change at any time. Feedback through our GitHub issues is appreciated
 - [Introduction](#introduction)
 - [Resource Object Schemas](#resource-object-schemas)
 - [API Endpoints](#api-endpoints)
-  - [Bundle Actions API](#bundle-actions-api)
```
Comment (teetone) [inline docs/REST-API-Reference.md]: I'm not sure why this change from https://github.com/codalab/codalab-worksheets/pull/2838 is also in this PR?

File: scripts/create_sample_worksheet.py [hunk 4]
```diff
@@ -20,16 +20,17 @@ class SampleWorksheet:
     _WORKSHEET_FILE_PATH = '/tmp/sample-worksheet-temp.txt'
```
Comment (teetone) [inline scripts/create_sample_worksheet.py]: I also see this change in another PR.

**Review by teetone — COMMENTED** [review]
It seems like there are some overlapping changes with the previous PR's. I can take another look after you merge the other PR's in first.

File: scripts/create_sample_worksheet.py (hunk 4 above)
Comment (yiboliu) [reply scripts/create_sample_worksheet.py]: It's because this PR is stacked on that one. I'm opening a PR per commit.

File: docs/REST-API-Reference.md (hunk 3 above)
Comment (yiboliu) [reply docs/REST-API-Reference.md]: It's because this PR is stacked on that one. I'm opening a PR per commit.

File: alembic/versions/2020081018_added_group_uuid_to_worker_9d3ab0b9d304.py (hunk 2 above)
Comment (yiboliu) [reply alembic/versions/2020081018_added_group_uuid_to_worker_9d3ab0b9d304.py:6]: I asked the question in the channel. Needs some further investigation.

File: alembic/versions/2014082017_add_worker_action_e_g__558ac277c2b9.py (hunk 1 above)
Comment (yiboliu) [reply alembic/versions/2014082017_add_worker_action_e_g__558ac277c2b9.py:19]: sure, I can do it.

File: alembic/versions/2020081018_added_group_uuid_to_worker_9d3ab0b9d304.py (hunk 2 above)
Comment (teetone) [reply alembic/versions/2020081018_added_group_uuid_to_worker_9d3ab0b9d304.py:6]: Can you create an issue for it for now?

File: docs/REST-API-Reference.md (hunk 3 above)
Comment (teetone) [reply docs/REST-API-Reference.md]: I thought we discussed having multiple PR's with different changes. This makes reviewing more difficult.

Comment (yiboliu) [reply docs/REST-API-Reference.md]: Oh, you can just go to the last commit of each PR and just review that commit. All previous commits are on other PRs.

File: alembic/versions/2020081018_added_group_uuid_to_worker_9d3ab0b9d304.py (hunk 2 above)
Comment (yiboliu) [reply alembic/versions/2020081018_added_group_uuid_to_worker_9d3ab0b9d304.py:6]: sure.

Comment (yiboliu) [reply alembic/versions/2020081018_added_group_uuid_to_worker_9d3ab0b9d304.py:6]: https://github.com/codalab/codalab-worksheets/issues/2896

File: alembic/versions/2014082017_add_worker_action_e_g__558ac277c2b9.py (hunk 1 above)
Comment (yiboliu) [reply alembic/versions/2014082017_add_worker_action_e_g__558ac277c2b9.py:19]: done

**Review by teetone — APPROVED** [review]
